src/compton.c

```diff
@@ -3530,12 +3530,9 @@ register_cm(session_t *ps) {
 
   // Set _NET_WM_PID
   {
-    long pid = getpid();
-    if (!XChangeProperty(ps->dpy, ps->reg_win,
-          get_atom(ps, "_NET_WM_PID"), XCB_ATOM_CARDINAL, 32, PropModeReplace,
-          (unsigned char *) &pid, 1)) {
-      printf_errf("(): Failed to set _NET_WM_PID.");
-    }
+    uint32_t pid = getpid();
+    xcb_change_property(c, XCB_PROP_MODE_REPLACE, ps->reg_win,
```

Comment:
Any reason to remove the error reporting code?

Reply:
From the commit message:

>     I removed the error checking in compton.c because it was dead code.
>    XChangeProperty() always returns 1.

And from debian codesearch: https://sources.debian.org/src/libx11/2:1.6.6-1/src/ChProp.c/?hl=33#L93